charts/quickwit/templates/metastore-deployment.yaml

```diff
@@ -38,13 +38,21 @@ spec:
       serviceAccountName: {{ include "quickwit.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      {{- with .Values.metastore.initContainers -}}
```

Comment:
To replace with `{{- with .Values.metastore.initContainers }}` (without the last `-`). Because there's still an error otherwise:

```shell
helm template . --values values.yaml --values values2.yaml
```

Here's the `values2.yaml` to reproduce the bug:

```yaml
---
configLocation: /tmp/yo.yml
config:
  default_index_root_uri: s3://qw-indexes
  storage:
    s3:
      endpoint: https://fr-par.scw.cloud
      region: fr-par
      access_key_id: SCWXXXXXXXX
      secret_access_key: XXXXXXXX

environment:
  QW_METASTORE_URI: s3://qw-indexes

indexer:
  initContainers: &initContainers
    - name: hello
      image: busybox:1.28
      command: ['echo', 'heyyyyy']
    - name: config
      image: busybox:1.28
      command: ['cp', '/tmp/yo.yaml', '/quickwit/node.yaml']

searcher:
  initContainers: *initContainers

janitor:
  initContainers: *initContainers

control_plane:
  initContainers: *initContainers

metastore:
  initContainers: *initContainers
```

Reply:
I fixed it now. My tests did not have initContainers set on all of the components. Apologies.

Reply:
No problem, I'll continue the tests ASAP :)

Thanks for the fix

Reply:
Fix ok:

```shell
$ helm template . --values values.yaml --values values2.yaml|head -2
---
# Source: quickwit/templates/serviceaccount.yaml
```

I'll perform some tests on kind today